### ISSUE TITLE
When a disconnected client connects, it should get sync. Should client pull request, or server push?

### ISSUE THREAD

**Issue by mmedlin1997** [chaisync/chai-server-socketio#2]
When a client connects, there is no mechanism to request an update from the server. 
Maybe in the 'on connect':
if client has new data during off-line, then emit full reminder JSON data
else emit {user, device} to announce user connection

**Comment by leo195162839** [comment]
Yeah, that's a good idea. If possible, I think we should let client send either local data or device and user info to server once hit the toggle connection button, then server respond with information.
